pages/linux/tree.md

```diff
@@ -29,3 +29,7 @@
 - Ignore entries that match a wildcard (glob) pattern:
 
 `tree -I {{*.txt}}`
+
+- Print the tree with ignoring of specific directories:
```

Comment:
I like your addition but I feel like it could be worded differently.
Maybe:
```suggestion
- Print the tree ignoring the given directories:
```

Reply:
nice! Will fix

Reply:
Fixed